packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -213,7 +214,20 @@ module.exports = {
     // to restart the development server for Webpack to discover it. This plugin
     // makes the discovery automatic so you don't have to restart.
     // See https://github.com/facebookincubator/create-react-app/issues/186
-    new WatchMissingNodeModulesPlugin(paths.appNodeModules)
+    new WatchMissingNodeModulesPlugin(paths.appNodeModules),
+    // Trigger some typechecking if a file matches with an @ flow comment
+    new FlowTypecheckPlugin({
+      flowconfig: [
+        '[ignore]',
+        '<PROJECT_ROOT>/node_modules/fbjs/.*',
```

Comment:
After https://github.com/facebook/fbjs/pull/182 (which was in [fbjs 0.8.5](https://github.com/facebook/fbjs/commits/v0.8.5)) this shouldn't be necessary! And from testing out create-react-app right now, it seems to install 0.8.6 :)